pkgs/development/libraries/libidn2/hack.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, libidn2, libunistring, runCommandLocal, patchelf }:
+# Construct a copy of libidn2.* where all (transitive) libc references (in .bin)
+# get replaced by a new one, so that there's no reference to bootstrap tools.
+runCommandLocal
+  "${libidn2.pname}-${libidn2.version}"
+  {
+    outputs = [ "bin" "dev" "out" ];
+    buildInputs = [ patchelf ];
```

Comment:
```suggestion
    nativeBuildInputs = [ patchelf ];
```

Reply:
Isn't patchelf even part of the stdenv or are we still before that?

Reply:
Right, dropped.  I think I sometime got burnt by this on darwin, but this hack is linux-only. (glibc-stdenv, but that's the same set as linux in our case).